extensions/ql-vscode/src/debugger/debug-configuration.ts

```diff
@@ -35,6 +36,15 @@ export type QLDebugConfiguration = DebugConfiguration & QLDebugArgs;
 export type QLResolvedDebugConfiguration = DebugConfiguration &
   CodeQLProtocol.LaunchConfig;
 
+/** If the specified value is a single element, then turn it into an array containing that element. */
+function makeArray<T extends Exclude<any, any[]>>(value: T | T[]): T[] {
```

Comment:
Curious...why `extends Exclude<any, any[]>`?

Reply:
Otherwise `makeArray<number[]>([ 5, 6, 7 ])` wouldn't know whether to return the input or wrap it in an array without some deeper runtime type inspection.